content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all external services by default
+
+An alternative to calling external services directly is to instruct the Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
```

Comment:
We should spend a sentence to explain why a user wants to do this, e.g. to quickly onboard their app to istio and worry about controlling access to external services later on.

Reply:
Yep, I added this sentence: 
> This option allows you to start evaluating Istio quickly, without controlling access to external
services, and decide to [configure access to external services](#configuring-istio-external-services) later.